BrainPortal/app/assets/stylesheets/neurohub.scss.erb

```diff
@@ -2092,6 +2093,44 @@ Secondary navbar with application-wide search + breadcrumbs
   }
 }
 
+/*
+=============================================================
+| COOKIE NOTIFICATION
```

Comment:
what the difference from COOKIE NOTIFICATION and Cookie Notification Panel should not both have same name?

Reply:
It's just a label for a comment section, see at the very top how Candice labels these sections.